Testscripts/Windows/STOR-ExportImportVM.ps1

```diff
@@ -162,17 +162,26 @@ function Main {
     foreach ($Vm in $VMs) {
         if ($ExportedVMID -ne $($Vm.VMId)) {
             $ImportedVM = $Vm.VMId
-            Get-VM -Id $Vm.VMId | Rename-VM -NewName $newName
+            Get-VM -Id $Vm.VMId -ComputerName $HvServer | Rename-VM -NewName $newName -Passthru
             break
         }
     }
 
-    Get-VMSnapshot -VMName $newName -ComputerName $HvServer -Name "TestExport" | Restore-VMSnapshot -Confirm:$False -Verbose
+    Restore-VMSnapshot -VMName $newName -ComputerName $HvServer -Name "TestExport" -Confirm:$False -Verbose 
```

Comment:
one more space at the end, please remove it
```
2019-08-22T08:20:09.0823385Z RuleName                            Severity     ScriptName Line  Message                                              
2019-08-22T08:20:09.0828991Z --------                            --------     ---------- ----  -------                                              
2019-08-22T08:20:09.0857450Z PSAvoidTrailingWhitespace           Information  STOR-Expor 170   Line has trailing whitespace                         
2019-08-22T08:20:09.0858316Z                                                  tImportVM.                                                            
2019-08-22T08:20:09.0859383Z                                                  ps1    
```

Reply:
Thank you Lili's help to review, the space is removed.

Best Regards,
Xuemin